tests/config/with_openai_api_key.yaml

```diff
@@ -0,0 +1,16 @@
+---
+llm_providers:
+  - name: openai
+    url: "https://api.openai.com/v1"
+    credentials_path: tests/config/secret.txt
+    models:
+      - name: ibm/granite-13b-chat-v1
```

Comment:
it doesn't really matter since it's just for a test, but to avoid confusion please use openai model names when defining openai providers (e.g. gpt35 in this case) because as written it makes me wonder if you're trying to test a case where someone accidentally mixed up a model from a different provider.